share/picongpu/tests/CollisionsThermalisation/cmakeFlags

```diff
@@ -29,22 +29,26 @@
 #   - start with zero index
 #   - increase by 1, no gaps
 
-# constant log
 flags[0]="-DPARAM_OVERWRITES:LIST='-DPARAM_START_IONS=Corner5000ppc;-DPARAM_START_ELECTRONS=Corner5000ppc'"
-flags[1]="-DPARAM_OVERWRITES:LIST='-DPARAM_START_IONS=Corner1000ppc;-DPARAM_START_ELECTRONS=Corner5000ppc'"
-flags[2]="-DPARAM_OVERWRITES:LIST='-DPARAM_START_IONS=Corner5000ppc;-DPARAM_START_ELECTRONS=Corner1000ppc'"
-# dynamic  log (both)
-flags[3]="-DPARAM_OVERWRITES:LIST='-DPARAM_START_IONS=Corner5000ppc;-DPARAM_START_ELECTRONS=Corner5000ppc;-DPARAM_DYNAMIC_LOG_INTER=1;-DPARAM_DYNAMIC_LOG_INTRA=1'"
-flags[4]="-DPARAM_OVERWRITES:LIST='-DPARAM_START_IONS=Corner1000ppc;-DPARAM_START_ELECTRONS=Corner5000ppc;-DPARAM_DYNAMIC_LOG_INTER=1;-DPARAM_DYNAMIC_LOG_INTRA=1'"
-flags[5]="-DPARAM_OVERWRITES:LIST='-DPARAM_START_IONS=Corner5000ppc;-DPARAM_START_ELECTRONS=Corner1000ppc;-DPARAM_DYNAMIC_LOG_INTER=1;-DPARAM_DYNAMIC_LOG_INTRA=1'"
-# dynamic  log (only inter)
-flags[6]="-DPARAM_OVERWRITES:LIST='-DPARAM_START_IONS=Corner5000ppc;-DPARAM_START_ELECTRONS=Corner5000ppc;-DPARAM_DYNAMIC_LOG_INTER=1;-DPARAM_DYNAMIC_LOG_INTRA=0'"
-flags[7]="-DPARAM_OVERWRITES:LIST='-DPARAM_START_IONS=Corner1000ppc;-DPARAM_START_ELECTRONS=Corner5000ppc;-DPARAM_DYNAMIC_LOG_INTER=1;-DPARAM_DYNAMIC_LOG_INTRA=0'"
-flags[8]="-DPARAM_OVERWRITES:LIST='-DPARAM_START_IONS=Corner5000ppc;-DPARAM_START_ELECTRONS=Corner1000ppc;-DPARAM_DYNAMIC_LOG_INTER=1;-DPARAM_DYNAMIC_LOG_INTRA=0'"
+flags[1]="-DPARAM_OVERWRITES:LIST='-DPARAM_START_IONS=Corner5000ppc;-DPARAM_START_ELECTRONS=Corner5000ppc;-DPARAM_DYNAMIC_LOG_INTER=1;-DPARAM_DYNAMIC_LOG_INTRA=1'"
+flags[2]="-DPARAM_OVERWRITES:LIST='-DPARAM_START_IONS=Corner5000ppc;-DPARAM_START_ELECTRONS=Corner5000ppc;-DPARAM_DYNAMIC_LOG_INTER=1;-DPARAM_DYNAMIC_LOG_INTRA=0'"
 # test support for 32bit precision
-flags[9]="-DPARAM_OVERWRITES:LIST='-DPARAM_START_IONS=Corner5000ppc;-DPARAM_START_ELECTRONS=Corner1000ppc;-DPARAM_COLLISION_PRECISION=float_32;-DPARAM_PRECISION=precision32Bit'"
+flags[3]="-DPARAM_OVERWRITES:LIST='-DPARAM_START_IONS=Corner5000ppc;-DPARAM_START_ELECTRONS=Corner1000ppc;-DPARAM_COLLISION_PRECISION=float_32;-DPARAM_PRECISION=precision32Bit'"
 # test support for mixed precision
-flags[10]="-DPARAM_OVERWRITES:LIST='-DPARAM_START_IONS=Corner5000ppc;-DPARAM_START_ELECTRONS=Corner1000ppc;-DPARAM_PRECISION=precision32Bit'"
+flags[4]="-DPARAM_OVERWRITES:LIST='-DPARAM_START_IONS=Corner5000ppc;-DPARAM_START_ELECTRONS=Corner1000ppc;-DPARAM_PRECISION=precision32Bit'"
```

Comment:
At first I thought the comments are mixed up but I'm not so sure. Maybe for clarity one could make the `PARAM_COLLISION_PRECISION` explicit in both cases.